luigi/contrib/s3.py

```diff
@@ -243,24 +245,23 @@ def get_key(self, path):
     def put(self, local_path, destination_s3_path, **kwargs):
         """
         Put an object stored locally to an S3 path.
-
+        :param local_path: Path to source local file
+        :param destination_s3_path: URL for target S3 location
         :param kwargs: Keyword arguments are passed to the boto function `put_object`
         """
-        if 'encrypt_key' in kwargs:
-            raise DeprecatedBotoClientException(
-                'encrypt_key deprecated in boto3. Please refer to boto3 documentation for encryption details.')
+        self._check_deprecated_argument(kwargs)
 
         # put the file
         self.put_multipart(local_path, destination_s3_path, **kwargs)
 
     def put_string(self, content, destination_s3_path, **kwargs):
         """
         Put a string to an S3 path.
+        :param content: Object data
```

Comment:
it should already be clear by the method name, but could we `:param content: Data str` or something. `Object data` just makes me hesitate.